c2corg_api/models/document.py

```diff
@@ -304,6 +308,29 @@ def to_archive(self):
     def update(self, other):
         copy_attributes(other, self, DocumentGeometry._ATTRIBUTES)
 
+    def almost_equals(self, other):
+        g1 = None
+        if isinstance(self.geom, geoalchemy2.WKBElement):
+            g1 = geoalchemy2.shape.to_shape(self.geom)
+        else:
+            # WKT are used in the tests.
+            str1 = string.split(self.geom, ';')[1]
+            g1 = wkt.loads(str1)
+
+        g2 = None
+        if isinstance(other.geom, geoalchemy2.WKBElement):
+            g2 = geoalchemy2.shape.to_shape(other.geom)
+        else:
+            # WKT are used in the tests.
+            str2 = string.split(other.geom, ';')[1]
+            g2 = wkt.loads(str2)
+
+        # FIXME: using almost_equals with +- 0.8m
+        # OK for EPSG:3857 but what about EPSG:4326?
```

Comment:
4326 is only used in the UI forms (but then converted back to 3857 when writing the request, as done in your last PR) or in the view pages. Only 3857 is passed to the API.